src/components/Info.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+class Info extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            isFav: false,
+            favImage: ''
+        };
+        this.handleClickFavourite = this.handleClickFavourite.bind(this);
+    }
+
+    componentDidMount() {
+        console.log("Info componentDidMount")
+        if (this.props.movieIsfav === true) {
+            document.querySelector(".fav-btn").src = "src/images/fav.png";
```

Comment:
IT would be better to handle this in render method. Since component may re-render at any time, doing direct DOM manipulation can result in unexpected behaviour